src/pages/productListing/productlisting.css

```diff
@@ -0,0 +1,65 @@
+.product-sidebar {
+    background-color: var(--primary-color);
+    color: var(--light-color);
+    grid-column-start: 1;
+    grid-column-end: 2;
```

Comment:
Can use grid-column shorthand property. grid-column:1/2

Reply:
but its's quite understandable to me that is why